src/bokeh/util/strings.py

```diff
@@ -60,14 +60,14 @@ def indent(text: str, n: int = 2, ch: str = " ") -> str:
     return "\n".join(padding + line for line in text.split("\n"))
 
 
-def nice_join(seq: Iterable[str], sep: str = ", ", conjuction: str = "or") -> str:
+def nice_join(seq: Iterable[str], sep: str = ", ", conjunction: str = "or") -> str:
```

Comment:
technically speaking a breaking change but not a very user-important function so we can let it slide. But as long as we are updating this I'd make the last two args be kwarg-only at this time:
```suggestion
def nice_join(seq: Iterable[str], *, sep: str = ", ", conjunction: str = "or") -> str:
```